uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/calendar-ctrl.js

```diff
@@ -358,6 +352,19 @@ define([
       }.bind(this));
     };
 
+    /**
+     * Returns leave status value from name
+     * @param {String} name - name of the leave status
+     * @returns {int/boolean}
+     */
+    function getLeaveStatusValuefromName(name) {
+      var leaveStatus = _.find(leaveRequestStatuses, function (status) {
+        return status.name === name;
+      });
+
+      return leaveStatus ? leaveStatus.value : false;
```

Comment:
When `leaveStatus` is not present, something needs to be returned, it will be either false/undefined/null. Returned false so that I can use ternary operator instead of an if condition.